slides/intro-03-what-makes-a-model.qmd

```diff
@@ -199,127 +209,114 @@ countdown::countdown(minutes = 5, id = "explore-tree-spec")
 
 ## Models we'll be using today
 
-* Logistic regression
-* Decision trees
+* Proportional hazards (PH) model
+* Decision tree
 
-```{r sim-model-viz}
+```{r data-model-illustration}
 #| echo: false
-
-set.seed(1)
-dat <- sim_logistic(500, ~ .1 + 2 * A)
-dat$bin <- cut(dat$A, breaks = c(seq(-3, 3, by = 1/2)), include.lowest = TRUE)
-bin_midpoints <- data.frame(A = seq(-3, 3, by = 1/2) + 0.25)
-
-rates <- 
-  dat %>% 
-  nest(.by = bin) %>% 
-  mutate(
-    probs = map(data, ~ binom.test(sum(.x$class == "one"), nrow(.x))),
-    probs = map(probs, ~ tidy(.x))
-  ) %>% 
-  select(-data) %>% 
-  unnest(cols = probs) %>% 
-  arrange(bin) %>% 
-  mutate(A = seq(-3, 3, by = 1/2) + 0.25) 
-
-plot_rates <- left_join(rates, bin_midpoints, by = join_by(A)) %>% 
-  filter(-2.5 < A, A < 3) %>% 
-  ggplot() + 
-  geom_point(aes(A, estimate)) +
-  geom_errorbar(aes(A, estimate, ymin = conf.low, ymax = conf.high), width = .25)  +
-  xlim(c(-3, 3.5)) +
-  theme_bw(base_size = 18)
+cat_new <- tibble(
+  intake_condition = factor(c("normal", "fractious", "feral", "other", "under_age_or_weight", "ill_moderatete", "ill_mild"))
+)
+  #theme_bw(base_size = 18)
 ```
 
-## Logistic regression
+## Proportional hazards model
 
 ::: columns
-::: {.column width="60%"}
-```{r plot-rates}
+::: {.column width="50%"}
+```{r plot-baseline}
 #| echo: false
 #| fig.width: 8
 #| fig.height: 7
 
-plot_rates
+fit(proportional_hazards(), event_time ~ 1, data = cat_train) %>% 
+  augment(new_data = cat_new[1,], eval_time = seq(0, 360, 1)) %>% 
+  tidyr::unnest(cols = .pred) %>% 
+  ggplot() +
+  geom_step(aes(.eval_time, .pred_survival)) +
```

Comment:
For smooth models we can use line plots instead of step plots (that can also help visually differentiate the two models later).

Reply:
Also, do you think we should use the column names as the labels (as opposed to "Evaluation time" etc)? 

We haven't discussed evaluation time yet (right?) so we might want to pull the few slides from #8 into this section?

Reply:
Smooth lines and different axis labels are great points, thanks! I think the introduction to the models works without dipping into eval time. It's part of the prediction section but only in terms of "survival probability at what time?". I haven't added more here because it seemed more natural to me to talk about it in the context of metrics (especially with the name "evaluation time")